library/telegraf

```diff
@@ -1,14 +1,7 @@
 Maintainers: David Reimschussel <dreimschussel@influxdata.com> (@reimda),
              Steven Soroka <ssoroka@influxdata.com> (@ssoroka)
 GitRepo: git://github.com/influxdata/influxdata-docker
-GitCommit: 8e5d747ca99f0db3327e64fd68a0690aa26b4ecd
-
-Tags: 1.16, 1.16.3
-Architectures: amd64, arm32v7, arm64v8
-Directory: telegraf/1.16
-
-Tags: 1.16-alpine, 1.16.3-alpine
-Directory: telegraf/1.16/alpine
+GitCommit: bcbbcef09c1a3f93ecf02e95e953e54006e7a0de
```

Comment:
This commit to `influxdata/influxdata-docker` is missing the `entrypoint.sh` for both the `1.19` directory and the `1.19/alpine` directory.